Source/Core/Core/PowerPC/BreakPoints.h

```diff
@@ -45,12 +47,21 @@ struct TWatch
   bool is_enabled = false;
 };
 
+enum class MemCheckCondition
+{
+    MEMCHECK_NONE = 0x00,
```

Comment:
This should adhere to the casing that @leoetlino used in his example [here](https://github.com/dolphin-emu/dolphin/pull/6325#discussion_r163789901). With a strongly-typed enum there's no need to repeat part of the name of the enum itself in its members (since the name itself already acts as a namespace).

Reply:
Ah, holdover from pcsx2.  Been looking at it so long did not even notice.  Will fix.